app/Controllers/Home.php

```diff
@@ -27,13 +25,123 @@ public function CreateEmployee()
     }
 
     public function BranchMaster()
     {
-        return view("branch-master");
+        $title = [
+            'page_title' => "Manage Branches",
+            'head' => "Branch",
+            'link' => 'Add Branch'
+        ];
+        $data = [
+            'active' => 'active-list',
+        ];
+        return view("branch-master", array_merge($data, $title));
+    }
+
+    public function Job()
+    {
+        $data = [
+            'active' => 'active-list',
+            'page_title' => "Manage Jobs",
+            'head' => 'Job',
+            'link' => 'Add job'
+        ];
+        return view('job-type', $data);
+    }
+
+    public function Deduction()
+    {
+        $data = [
+            'active' => 'active-list',
+            'page_title' => "Manage Deduction Types",
+            'head' => 'Deductions',
+            'link' => 'Add deduction'
+        ];
+        return view('deductions', $data);
+    }
+
+    public function Contract()
+    {
+        $data = [
+            'active' => 'active-list',
+            'page_title' => "Manage Contract Type",
+            'head' => 'Contracts',
+            'link' => 'Add contracts'
+        ];
+        return view('contract', $data);
+    }
+
+    public function Payement()
+    {
+        $data = [
+            'active' => 'active-list',
+            'page_title' => "Manage Payement Method",
+            'head' => 'Payements',
+            'link' => 'Add payement method'
+        ];
+        return view('payement',$data);
     }
 
-    
+    public function Designation()
+    {
+        $data = [
+            'active' => 'active-list',
+            'page_title' => "Manage Designation",
+            'head' => 'Designation',
+            'link' => 'Add designation'
+        ];
+        return view('designation',$data);
+    }
+
+    public function Leave()
+    {
+        $data = [
+            'active' => 'active-list',
+            'page_title' => "Manage Leave Types",
+            'head' => 'Leaves',
+            'link' => 'Add leave type'
+        ];
+        return view('leave', $data);
+    }
+    public function Loan()
+    {
+        $data = [
+            'active' => 'active-list',
+            'page_title' => "Manage Loans",
+            'head' => 'Loans',
+            'link' => 'Add loan'
+        ];
+        return view('loan',$data);
+    }
+    public function Shift()
+    {
+        $data = [
+            'active' => 'active-list',
+            'page_title' => "Manage Shift Types",
+            'head' => 'Shifts',
+            'link' => 'Add shift type'
+        ];
+        return view('shift',$data);
+    }
+
+
+    public function Permission()
+    {
+        $title = [
+            'page_title' => "Manage Permissions",
+            'head' => "Permissions",
+            'link' => "Asign Permission"
+        ];
+        $data = [
+            'active' => 'active-list',
+        ];
+        return view('permission', array_merge($data, $title));
+    }
+        
 }
```

Comment:
rewrite functions to fetch data and work on permissions

Reply:
Am not understanding well what you are trying to say